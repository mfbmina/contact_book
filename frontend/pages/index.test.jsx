import { shallow } from 'enzyme';
import React from 'react';

import Home from './index';

describe('Index page', () => {
  it('should render', () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
