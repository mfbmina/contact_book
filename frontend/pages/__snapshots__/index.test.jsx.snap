// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should render 1`] = `ShallowWrapper {}`;
